### PR TITLE
Teams fx migration samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Review by formulahendry — APPROVED** [review]

File: samples/meeting-recruitment-app/nodejs/Readme.md [hunk 1]
```diff
@@ -40,6 +40,20 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 - [NodeJS](https://nodejs.org/en/) must be installed on your development machine (version 16.14.2  or higher).
 - [ngrok](https://ngrok.com/download) or equivalent tunnelling solution
 - [Teams](https://teams.microsoft.com) Microsoft Teams is installed and you have an account
+- [Teams Toolkit for VS Code](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) or [TeamsFx CLI](https://learn.microsoft.com/microsoftteams/platform/toolkit/teamsfx-cli?pivots=version-one)
+
+## Run the app (Using Teams Toolkit for Visual Studio Code)
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio Code.
+
+1. Ensure you have downloaded and installed [Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
```
Comment (ChetanSharma-msft) [inline samples/meeting-recruitment-app/nodejs/Readme.md:49]: What about deployment steps?

File: samples/meeting-recruitment-app/nodejs/appPackage/manifest.json [hunk 2]
```diff
@@ -12,20 +12,21 @@
...
-    "outline": "icon-outline.png",
-    "color": "icon-color.png"
+    "outline": "outline.png",
+    "color": "color.png"
   },
   "accentColor": "#ffffff",
   "configurableTabs": [
     {
-      "configurationUrl": "<<BASE-URL>>/configure",
+      "configurationUrl": "https://${{BOT_DOMAIN}}:3000/configure",
```
Comment (ChetanSharma-msft) [inline samples/meeting-recruitment-app/nodejs/appPackage/manifest.json:29]: Why we required 3000??

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added few comments

File: samples/meetings-token-app/nodejs/Readme.md [hunk 3]
```diff
@@ -57,12 +57,6 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 ### Tools
 
-- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
-  ```bash
-  # determine dotnet version
-  dotnet --version
-  ```
-
 - [Nodejs](https://nodejs.org/en/download/) version 10.21.0+ (use the LTS version)
```
[the remark this answers is not in the document]
Comment (Harikrishnan-MSFT) [reply samples/meetings-token-app/nodejs/Readme.md:60]: Fixed

File: samples/meeting-recruitment-app/nodejs/Readme.md (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/meeting-recruitment-app/nodejs/Readme.md:49]: Fixed

File: samples/meeting-recruitment-app/nodejs/Readme.md [hunk 4]
```diff
@@ -81,6 +95,11 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
     ```bash
     ngrok http 3978 --host-header="localhost:3978"
     ```
+    Alternatively, you can also use the `dev tunnel`. Please follow [Create and host a Dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:
```
Comment (ChetanSharma-msft) [inline samples/meeting-recruitment-app/nodejs/Readme.md:98]: Need to update this contents.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comment

**Review by Mohammed-MSFT — APPROVED** [review]
Changes Looks Fine, Approving It.

File: samples/meeting-recruitment-app/nodejs/Readme.md (hunk 4 above)
Comment (Harikrishnan-MSFT) [reply samples/meeting-recruitment-app/nodejs/Readme.md]: Fixed

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline -** 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5962805228

image

**Review by Prithvi-MSFT — APPROVED** [review]

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**